dist/notice/ds4/notice.css

```diff
@@ -306,7 +306,7 @@ button.page-notice__close span {
   height: 0;
   position: absolute;
   width: 0;
-  left: calc(42%);
```

Comment:
Looks like this fixed the `calc` bug, however I'm not sure if `42%` was intended here or actually `50% - 8px`. Assuming the latter.